sdk/highlight-ruby/highlight/lib/highlight.rb

```diff
@@ -57,9 +57,6 @@ def trace(session_id, request_id, attrs = {})
         HIGHLIGHT_TRACE_ATTRIBUTE => request_id
       }.merge(attrs).compact) do |_span|
         yield
-      rescue StandardError => e
-        record_exception(e)
```

Comment:
ah, so the otel instrumentation already automatically reports the exception event when one is raised, so we don't need to explicitly create another span

Reply:
Yeah, exactly!